stable/datacube-dashboard/templates/install-postgis.yaml

```diff
@@ -19,6 +19,11 @@ spec:
         release: {{.Release.Name | quote }}
         chart: "{{.Chart.Name}}-{{.Chart.Version}}"
     spec:
+      # Turning single-request-reopen option on would fix issue where in two requests from the same port are
+      # not handled correctly it will close the socket and open a new one before sending the second request.
+      dnsConfig:
+        options:
+          - name: single-request-reopen
```

Comment:
I don't think this is required while installing database changes

Reply:
Removed from `install-postgis.yaml` file